imperial_coldfront_plugin/views.py

```diff
@@ -592,6 +605,14 @@ def add_rdf_storage_allocation(request):
             )
             messages.success(request, "RDF allocation created successfully.")
 
+            if settings.GPFS_ENABLED:
+                gpfs_create_fileset_in_background(
+                    filesystem_name, fileset_name, owner_id, group_id, path, permissions
+                )
+                gpfs_set_quota_in_background(
+                    filesystem_name, fileset_name, block_quota, files_quota
+                )
```

Comment:
We'll need to create the fileset and set the quota as part of the same background task. They can't be separate tasks because we need to make sure the fileset creation completes successfully before setting the quota and getting that sort of control with background tasks is a bit elaborate.